src/pip/_internal/utils/compat.py

```diff
@@ -226,17 +190,6 @@ def expanduser(path):
            (sys.platform == 'cli' and os.name == 'nt'))
 
 
-def samefile(file1, file2):
-    # type: (str, str) -> bool
-    """Provide an alternative for os.path.samefile on Windows/Python2"""
-    if hasattr(os.path, 'samefile'):
-        return os.path.samefile(file1, file2)
-    else:
-        path1 = os.path.normcase(os.path.abspath(file1))
-        path2 = os.path.normcase(os.path.abspath(file2))
-        return path1 == path2
-
-
 if hasattr(shutil, 'get_terminal_size'):
```

Comment:
This is also a Py2 fallback, I think.

Reply:
👍 New in version 3.3.

https://docs.python.org/3/library/shutil.html#shutil.get_terminal_size